README.md

```diff
@@ -38,7 +38,7 @@ we would like things to flow.
 ### Requirements
 
 * [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
-* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
```

Comment:
Thanks for your contribution! Nice idea. But would you also remove the link? e.g. to do the same as we have in the [nexus-internal.](https://github.com/sonatype/nexus-internal)  --> `OpenJDK 8`

Also, please click the `sign the Sonatype Contributor License Agreement.` link in the `Conversation` tab in your PR. 
![image](https://user-images.githubusercontent.com/8549071/62934688-db7f2900-bdcd-11e9-8cc4-71fdb07537f8.png)